tests/templates/kuttl/smoke/00-limit-range.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: limit-request-ratio
+spec:
+  limits:
+  - type: "Container"
+    maxLimitRequestRatio:
+      cpu: 5
```

Comment:
Running this tests results in the following error:

> create Pod hello-world-server-default-0 in StatefulSet hello-world-server-default failed error: pods "hello-world-server-default-0" is forbidden: cpu max limit to request ratio per Container is 5, but provided ratio is 20.000000

Changing this to `cpu: 20` results in a successful test.

Reply:
Weird, tests passed on my machine...

Reply:
Nevermind, found the problem